app/provider.html

```diff
@@ -0,0 +1,351 @@
+  <!-- endbuild-->
+
+  <!-- build:js bower_components/webcomponentsjs/webcomponents-lite.min.js -->
+  <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+  <!-- endbuild -->
+
+  <!-- will be replaced with elements/elements.vulcanized.html -->
+  <link rel="import" href="elements/elements.html">
+  <!-- endreplace-->
+
+  <!-- For shared styles, shared-styles.html import in elements.html -->
+  <style is="custom-style" include="shared-styles"></style>
+  <style is="custom-style" include="entity-view-styles"></style>
+  <style is="custom-style" include="icon-styles"></style>
+  <style is="custom-style" include="map-styles"></style>
+
+</head>
+
+<body unresolved class="fullbleed layout vertical">
+  <span id="browser-sync-binding"></span>
+  <template is="dom-bind" id="app">
+
+    <!-- exposes clientConfig object which is a container for data 
+      transform functions -->
+    <client-config config="{{clientConfig}}"></client-config>
+    
+    <!--get app configuration from server -->
+    <iron-ajax
+      id="ajax"
+      auto
+      url="/config"
+      handle-as="json"
+      last-response="{{config}}"
+      on-response="showConfig">
+    </iron-ajax>
+     
+    <!--initialize new elastic client connection to server
+      esclient can be used throughout the application lifecycle -->
+    <elastic-client 
+      config="[[config.elasticConfig]]"
+      client="{{esclient}}">
+    </elastic-client>
+
+    <!-- Get the provider details -->
+    <query-builder 
+      query="{{providerQuery}}" 
+      query-template="{{config.queryTemplates.commonMatchQuery}}"
+      field="[[params.field]]" value="[[params.value]]">
+    </query-builder>
+
+    <elastic-search 
+      client="[[esclient]]"
+      index="[[config.elasticIndex]]"
+      query='[[providerQuery]]'
+      elastic-type='["adultservice"]'
+      results="{{provider}}"
+      lastError="{{error}}">
+    </elastic-search>
+
+    <json-transform
+     data-in="[[provider]]"
+     data-out="{{providerDisplay.provider}}"
+     transform-function="[[clientConfig.transforms.provider.provider]]"
+     ></json-transform>
+    
+    <!--
+    Get all Offers for this provider
+    -->
+    <query-builder 
+        query="{{providerOffersQuery}}" 
+        query-template="{{config.queryTemplates.commonMatchQuery}}"
+        field="itemOffered.uri" value="[[params.value]]">
+    </query-builder>
+
+    <elastic-search 
+      client="[[esclient]]"
+      index="[[config.elasticIndex]]"
+      query='[[providerOffersQuery]]'
+      results="{{providerOffers}}"
+      elastic-type='["offer"]'
+      lastError="{{error}}">
+    </elastic-search>
+
+    <json-transform
+     data-in="[[providerOffers]]"
+     data-out="{{providerDisplay.offers}}"
+     transform-function="[[clientConfig.transforms.relatedEntities.offer]]"
+     ></json-transform>
+
+     <!--
+        build the query for provider locations 
+      -->
+       <query-builder 
```

Comment:
While not strictly enforced, we would appreciate if you used consistent tabbing:  for example, compare the tabbing of lines 130-149 with the rest of the file.